tests/src/test/scala/actionContainers/ActionContainer.scala

```diff
@@ -52,6 +52,7 @@ import org.apache.openwhisk.core.containerpool.Container
 trait ActionContainer {
   def init(value: JsValue): (Int, Option[JsObject])
   def run(value: JsValue): (Int, Option[JsObject])
+  def runForJsArray(value: JsValue): (Int, Option[JsArray])
```

Comment:
Normally, need to change `def run(...)` as below
```
def run(value: JsValue): (Int, Option[JsValue])
```
But i just add another new method here, two reasons.
* If change `def run` directly, there has a lot of changes.
* It is just for test, so i just add another new method, there has no need to introduce too many code modifications